### PR TITLE
add v14 release docs

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
Update v14 release docs
- update `release-notes.md`
- update `data-files-description.md`
- update `data-formats.md`

#### What GitHub issue does your pull request address?
#432 

#### Which areas should receive a particularly close look?
Do the docs look sufficient/am I missing anything?

#### Is there anything that you want to discuss further?
I updated the links to the D3b workflow repo because we renamed it for consistency with other repos and to not mix other publication workflows with this one.

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
Yes for docs - want to get OK so we can add these to S3.
No for data download.

### Data Release Checklist
- [x] Is the table in doc/data-file-descriptions.md up to date?
- [x] Is doc/data-format.md up to date?
- [x] Is doc/release-notes.md up to date?
- [x] Is download-data.sh up to date? - yes, but folder does not yet exist

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
@jharenza I think you might as well fold in the changes/feedback from #456 here. `data-formats.md` does not get distributed so that won't hold anything up.

File: doc/data-files-description.md [hunk 1]
```diff
@@ -28,12 +25,14 @@ This document contains information about all data files associated with this pro
...
-|`intersect_exon_lancet_strelka_mutect_WGS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions
-|`intersect_exon_WXS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions
+|`intersect_cds_lancet_strelka_mutect_WGS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with Lancet, Strelka2, Mutect2 regions
+|`intersect_cds_WXS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions
```
Comment (cansavvy) [inline doc/data-files-description.md]:
```suggestion
|`intersect_cds_lancet_WXS.bed` | Analysis file | [`analyses/snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/snv-callers/) | Intersection of `gencode.v27.primary_assembly.annotation.gtf.gz` CDS with WXS 100bp padded BED regions and Lancet's regions
```

Comment (jharenza) [reply doc/data-files-description.md]: updated

**Comment by jharenza** [comment]
@jaclyn-taroni I will work on #456 tonight. The data is now on s3, and download confirmed to work. Implemented @cansavvy changes in the docs. Ready for review!

**Comment by cansavvy** [comment]
@jharenza See #512

**Comment by jharenza** [comment]
@jaclyn-taroni I updated the `data-formats.md` file with more GISTIC output info - I decided to only include potential relevant files that may be used, rather than all, and add some brief descriptions. Let me know if you would like more detail.

**Comment by jaclyn-taroni** [comment]
@jharenza - in https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/507/commits/c48f34056e00c8691371ee9e53342329834a633c, I added links to the HGG and non-MB, non-ATRT embryonal tumor subtyping analysis modules. Am I correct that these are the only molecular subgroup labels that were updated in this release?

**Comment by jharenza** [comment]
yes, correct!

File: doc/release-notes.md [hunk 2]
```diff
@@ -1,5 +1,118 @@
...
+### release-v14-20200203
+- release date: 2020-02-03
+- status: available
+- changes:
+  - Update kallisto stranded file to remove index column per [#474](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/474). Also removed samples from last re-sequencing polyA+stranded batch that were missed with the v13 release:
+    - pbta-gene-expression-kallisto.stranded.rds
+  - Update matrices of ependymonal tumor and embryonal tumor fusions of interest by biospecimen from [`analyses/fusion-summary`](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/478) to include all RNA biospecimens in new `pbta-histologies.tsv` file without fusion calls. Files updated:
+    - fusion_summary_embryonal_foi.tsv
+    - fusion_summary_ependymoma_foi.tsv
+  - Update strelka2 and mutect2 TCGA MAF files per [#483](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/483) and [#512](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/512). Files updated:
```
Comment (jaclyn-taroni) [inline doc/release-notes.md]:
```suggestion
  - Update Strelka2, Mutect2, and Lancet TCGA MAF files per [#483](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/483) and [#512](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/512). Files updated:
```

**Comment by jharenza** [comment]
We are re-running Lancet for TCGA, per the comment [here](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/512) - will update here once completed and updated on S3.

**Pull request merged**